src/ssl.c

```diff
@@ -16367,16 +16367,26 @@ WOLFSSL_ASN1_INTEGER* wolfSSL_X509_get_serialNumber(WOLFSSL_X509* x509)
 int wolfSSL_ASN1_TIME_print(WOLFSSL_BIO* bio, const WOLFSSL_ASN1_TIME* asnTime)
 {
     char buf[MAX_TIME_STRING_SZ];
+    int  ret = SSL_SUCCESS;
 
     WOLFSSL_ENTER("wolfSSL_ASN1_TIME_print");
 
     if (bio == NULL || asnTime == NULL)
         return BAD_FUNC_ARG;
```

Comment:
returning BAD_FUNC_ARG is against the openssl spec that says that 0 or 1 should be returned by this function.